src/js/view/captionsrenderer.js

```diff
@@ -273,8 +273,16 @@ const CaptionsRenderer = function (viewModel) {
 
     function _setShadowDOMFontSize(playerId, fontSize) {
         // Set Shadow DOM font size (needs to be important to override browser's in line style)
-        _windowStyle.fontSize = fontSize + 'px';
-        css('#' + playerId + ' .jw-video::-webkit-media-text-track-display', _windowStyle, playerId, true);
+        const selector = `#${playerId} .jw-video::-webkit-media-text-track-display`;
+        if (fontSize) {
+            fontSize += 'px';
+            if (OS.iOS) {
+                // Force layout after exiting fullscreen
```

Comment:
is there a way to scope this function to only run on fs exit?

Reply:
We could check if `fontSize` was set to `null` because of what happens before this function is run:
```
if (_model.get('fullscreen') && OS.iOS) {
     fontSize = null;
```
But I'd rather not add side-effects or pass in the model. Ensuring these styles update on both enter and exit of fullscreen is best.